src/lib/build.js

```diff
@@ -18,6 +18,7 @@ const getBuildOptions = ({
   telemetry: false,
   // buffer = true will not stream output
   buffer: json || silent,
+  offline,
```

Comment:
I don't think we actually need to pass `offline` to `@netlify/build` since we use `cachedConfig`.

I'm still doing it for clarify